src/components/ContactImportationModal/index.jsx

```diff
@@ -48,9 +48,15 @@ class ContactImportationModal extends React.Component {
     const finishedImportation = await finishedImportationPromise
 
     if (finishedImportation.status === Status.COMPLETE_SUCCESS) {
-      Alerter.success('importation.complete_success', {
-        smart_count: Importation.total(finishedImportation)
-      })
+      const created = Importation.created(finishedImportation)
+      const updated = Importation.updated(finishedImportation)
+      const { t } = this.context
+      Alerter.success(
+        `${t('importation.complete_success')}: ${t(
+          'importation.creation',
+          created
+        )}, ${t('importation.update', updated)}.`
+      )
```

Comment:
I had to update the notification message:

![notification message](https://user-images.githubusercontent.com/701648/44522508-0bbe4b00-a6d7-11e8-90ee-ce2fd45a19e4.gif)